src/kubernetes-ingress-controller/guides/using-kongplugin-resource.md

```diff
@@ -316,15 +382,16 @@ Server: gunicorn/19.9.0
 Access-Control-Allow-Origin: *
 Access-Control-Allow-Credentials: true
 demo:  injected-by-kong
-X-Kong-Upstream-Latency: 3
+X-Kong-Upstream-Latency: 2
 X-Kong-Proxy-Latency: 0
-Via: kong/1.2.1
+Via: kong/2.8.1
 ```
 
 ## Configure a global plugin
 
-Now, we will protect our Kubernetes cluster.
-For this, we will be configuring a rate-limiting plugin, which
+Instead of applying plugins to specific services or ingress routes,
+we can apply plugins to protect the entire gateway.
+To show this, we will be configuring a rate-limiting plugin, which
 will throttle requests coming from the same client.
```

Comment:
```suggestion
To test this, let's configure a rate-limiting plugin to throttle requests coming from the same client.
```
Trying to get rid of the future tense whenever possible (and "we", but that's a whole other issue). I know these guides are full of it, so it's a slow change.